admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java

```diff
@@ -109,13 +112,21 @@ public ApplicantDetailsDto getApplicantDetails(String rid) throws Exception {
             JSONObject idenitityJsonObject=objectMapper.readValue(idenitityJson,JSONObject.class);
             JSONObject mapperIdentity=utility.getJSONObject(idenitityJsonObject,IDENTITY);
             List<String> mapperJsonKeys = new ArrayList<>(mapperIdentity.keySet());
-            for(String valueObj: applicantDetails){
-                if(valueObj!=null && !valueObj.equalsIgnoreCase(ApplicantPhoto)){
+            for (String valueObj : applicantDetails) {
+                if (valueObj == null) {
+                    log.warn("Encountered null value in applicantDetails list");
+                    continue;
+                }
+                if (!valueObj.equalsIgnoreCase(ApplicantPhoto)) {
```

Comment:
"ApplicantPhoto" we should follow the naming convention here.

Reply:
changes as per the suggestion